Dockerfile

```diff
@@ -11,11 +11,14 @@ RUN mkdir -p /opt \
     && pip install --prefer-binary --no-cache-dir \
         poetry==1.2.2 \
     && poetry config virtualenvs.create false \
-    && poetry install --only main \
+    && poetry install --without dev \
+    && poetry build -f wheel \
     && apt-get purge -y -q g++ \
     && apt-get --purge -y -qq autoremove 
     
 # Stage: runtime
 FROM build AS runtime
-COPY ./labelmerge /opt/labelmerge
-ENTRYPOINT ["/opt/labelmerge/run.py"]
\ No newline at end of file
+RUN WHEEL=`ls /opt/labelmerge/dist/* | grep whl` \
```

Comment:
With the caveat that getting the path to the wheel might be trickier than this, the image would be a bit smaller if we did something like:

```
FROM python:3.9-slim-bullseye AS runtime
COPY --from=build /opt/labelmerge/dist/*.whl /opt/labelmerge
RUN WHEEL=`ls /opt/labelmerge | grep whl` \
    && pip install $WHEEL \
    && rm -r /opt/labelmerge
ENTRYPOINT ["labelmerge"]
```

Reply:
Can definitely give this a try on Monday, this container is easy and quick enough to build.